src/deploy/NVA_build/builder.Dockerfile

```diff
@@ -4,13 +4,16 @@ LABEL maintainer="Liran Mauda (lmauda@redhat.com)"
 ##############################################################
 # Layers:
 #   Title: Installing pre requirments
-#   Size: ~ 613 MB
+#   Size: ~ 1324 MB
```

Comment:
OMG ...
@liranmauda Are we going to survive this bump? (assuming it really bumped in this PR and not before)